aptos-move/aptos-release-builder/src/components/framework.rs

```diff
@@ -35,23 +43,41 @@ pub fn generate_upgrade_proposals(is_testnet: bool) -> Result<Vec<(String, Strin
             .unwrap()
             .to_string();
 
+        let mut args = vec![
+            "run",
+            "--bin",
+            "aptos",
+            "--",
+            "governance",
+            "generate-upgrade-proposal",
+            "--account",
+            publish_addr,
+            "--output",
+            move_script_path.to_str().unwrap(),
+            "--package-dir",
+            package_path.to_str().unwrap(),
+        ];
+
+        if is_testnet {
+            args.push("--testnet");
+        }
+
+        let mut _execution_hash: String = "".to_owned();
```

Comment:
nit: Remove _execution_hash => execution_hash

Reply:
I think `execution_hash` gives a compiler complaint bc if `next_execution_hash.is_empty()`, we wouldn't be using this value at all